src/hotspot/cpu/riscv/nativeInst_riscv.hpp

```diff
@@ -111,41 +111,52 @@ NativeCall* nativeCall_before(address return_address);
 // The NativeCall is an abstraction for accessing/manipulating native
 // call instructions (used to manipulate inline caches, primitive &
 // DSO calls, etc.).
+// NativeCall is reloc call on RISC-V. See MacroAssembler::reloc_call.
 class NativeCall: private NativeInstruction {
  // private: when common code is using byte_size()
  private:
   enum {
```

Comment:
I see the enum of NativeFarCall was named as `RISCV_specific_constants`, do we need this for NativeCall?

Reply:
Thank you having a look.

Seems not, there is no name for this enum in orignal code . And in this file some enums have names, some not, but seems either way is fine, although I think the names are all redundant here.